components/Buttons/examples/ButtonsContentEdgeInsetsExample.m

```diff
@@ -60,6 +61,7 @@ - (void)viewDidLoad {
   [self.floatingActionButton setContentEdgeInsets:UIEdgeInsetsMake(40, 40, 0, 0)
                                          forShape:MDCFloatingButtonShapeDefault
                                            inMode:MDCFloatingButtonModeNormal];
+  self.floatingActionButton.accessibilityLabel = @"Floating Button";
```

Comment:
We're all over the place with our accessibility labels, but I believe we should try to align on using "Sentence case" rather than "Title Case".